importer/RuRu.py

```diff
@@ -0,0 +1,107 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class RuRu(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Preferably try to resolve the 'district' (район/округ,
+        the field contains different types so that's why we're
+        matching it against the whole big list of administrative
+        divisions of different kinds).
+        If that doesn't work, use the higher level субъект.
```

Comment:
higher __region__ level ... (to connect to field name).

I see that there is both a `region` and a `region_iso` field. IS the second always used or is the former sometimes used instead?